databricks/koalas/tests/test_indexes.py

```diff
@@ -2003,3 +2003,32 @@ def test_astype(self):
 
         with self.assertRaisesRegex(TypeError, "not understood"):
             kidx.astype("int63")
+
+    def test_to_list(self):
+        # Index
+        pidx = pd.Index([1, 2, 3, 4, 5])
+        kidx = ks.from_pandas(pidx)
+        # MultiIndex
+        tuples = [(1, "red"), (1, "blue"), (2, "red"), (2, "green")]
+        pmidx = pd.MultiIndex.from_tuples(tuples)
+        kmidx = ks.from_pandas(pmidx)
+
+        # Only support as `to_list` in pandas < 0.24.0.
```

Comment:
Can we just compare the results with `tolist` too?

Reply:
Sounds good !